string task_name
geometry_msgs/Pose start_pose
geometry_msgs/Pose goal_pose
---
bool started